src/strategyHelpers/blueGreen/serviceBlueGreenHelper.ts

```diff
@@ -25,15 +25,15 @@ export async function deployBlueGreenService(
       GREEN_LABEL_VALUE
    )
 
-   // create other non deployment and non service entities
    const newObjectsList = manifestObjects.otherObjects
       .concat(manifestObjects.ingressEntityList)
       .concat(manifestObjects.unroutedServiceEntityList)
    const manifestFiles = fileHelper.writeObjectsToFile(newObjectsList)
-   if (manifestFiles.length > 0) await kubectl.apply(manifestFiles)
+   
```

Comment:
removing the length check here will make this throw an error if manifests is an empty array. Is that intended?

Reply:
that was not intended, thanks for catching this! not sure how that happened